CHANGES

```diff
@@ -1,3 +1,70 @@
+Release 1.6 (development)
+-------------------------
+
+.. rubric:: Enhancements
+
+* Added support for NOR flash devices. (by Ladislav Michl)
+* Added support for configuring the number of boot attempts for U-Boot. (by
+  Daniel Mack)
+* Implemented passing the image size to hooks as ``RAUC_IMAGE_SIZE``. (by Marcel
+  Hellwig)
+* Added support to use ``systemd.verity_root_data=`` to find the booted slot.
+  (by Arnaud Rebillout)
+* Implemented passing additional information to hooks for the ``boot-*`` slot types.
+  (by Bastian Krause)
+* Added support for extracting and replacing the bundle signature, which is
+  useful for scenarios with strict limitations on how HSMs can be used. (by
+  Jean-Pierre Geslin)
+* Implemented a ``rauc mount`` command to allow inspection of bundles without
+  extraction.
+* Allowed omitting the image filename when using the ``install`` slot hook.
+* Implemented support for extracting tar archives to jffs2 slots. (by Holger Assmann)
+* Added option to ignore expired certificates (``--no-check-time``). (by Michael Heimpold)
+* Added option to disable the concurrent access checks for plain bundles during
+  development (``--trust-environment``).
+* Simplified usage with squashfs image extensions as created by OpenEmbedded.
+  (by Omer Akram)
+
+.. rubric:: Bug fixes
```

Comment:
I found these two bug fixes from my pr #645 worth mentioning as I've seen myself and others hitting this multiple times:

* Fixed an issue with slots boot status determination that could accidentally mark 'good' slots as 'bad'
* Fixed emitting inconsistent slot status vie D-Bus API